src/main/java/com/mtab/spring/boot/service/BkStrServiceV1.java

```diff
@@ -0,0 +1,72 @@
+package com.mtab.spring.boot.service;
+
+import com.mtab.spring.boot.service.models.Book;
+import com.mtab.spring.boot.service.models.BookPurchase;
+import com.mtab.spring.boot.service.models.NewPrice;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class BkStrServiceV1 implements BkStrService{
+
+    private final Map<String, Book> books = new HashMap<>();
+    public final double centsToDollars = 0.01;
+
+    @Override
+    public int addBookToInventory(Book book) {
+        System.out.println(" ");
+        Book oldBook = books.get(book.bookID());
+        if(oldBook == null) {
+            books.put(book.bookID(), book);
+            System.out.println("New Book entered:");
+        } else {
+            books.put(book.bookID(), new Book(oldBook.bookID(), oldBook.title(), oldBook.author(), oldBook.price(), oldBook.numberInStock()+book.numberInStock()));
```

Comment:
It would be better to have this "increase quantity" method inside of Book, cleaner.

This approach is tightly coupling the service to the internals of the book class.